discovery-provider/src/models.py

```diff
@@ -603,3 +606,24 @@ def __repr__(self):
 tag={self.tag},\
 track_id={self.track_id},\
 owner_id={self.owner_id}>"
+
+class L2ContentNode(Base):
+    __tablename__ = "l2_content_nodes"
+    blockhash = Column(String, ForeignKey("blocks.blockhash"), nullable=False)
+    is_current = Column(Boolean, nullable=False)
+    cnode_id = Column(Integer, nullable=False)
+    delegate_owner_wallet = Column(String, nullable=False)
+    proposer_sp_ids = Column(postgresql.ARRAY(Integer), nullable=False)
+    proposer_1_address = Column(String, nullable=False)
+    proposer_2_address = Column(String, nullable=False)
+    proposer_3_address = Column(String, nullable=False)
+    created_at = Column(DateTime, nullable=False)
+
+    PrimaryKeyConstraint(is_current, cnode_id, blockhash)
+
+    def __repr__(self):
+        return f"<L2ContentNode(blockhash={self.blockhash},\
+is_current={self.is_current},cnode_id={self.cnode_id},\
+delegate_owner_wallet={self.delegate_owner_wallet}, proposer_sp_ids={self.proposer_sp_ids},\
+proposer_1_address={self.proposer_1_address},proposer_2_address={self.proposer_2_address},\
+proposer_3_address={self.proposer_3_address})>"
```

Comment:
I think we can use string addition or using `"""` to escape the entire string. This happens in other places too.